<?xml version="1.0"?>
<launch>
    <arg name="action_dispatch_topic" default="/kcl_rosplan/action_dispatch"/>
    <arg name="action_feedback_topic" default="/kcl_rosplan/action_feedback"/>

    <node pkg="mdr_find_object_action" type="find_object_client" name="find_object_client" output="screen">
        <param name="action_name" type="str" value="find_object" />
        <param name="server_name" type="str" value="find_object_server" />
        <param name="action_timeout" type="double" value="120" />
        <remap from="action_dispatch_topic" to="$(arg action_dispatch_topic)" />
        <remap from="action_feedback_topic" to="$(arg action_feedback_topic)" />
    </node>
</launch>
